conf.py

```diff
@@ -33,8 +33,6 @@
 lexers['php'] = PhpLexer(startinline=True)
 lexers['php-annotations'] = PhpLexer(startinline=True)
 
-primary_domain = 'php'
```

Comment:
we should use sphinx 1.5.* instead of 1.2 (I'll update installation docs of sphinx)